### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by midhunbiju**
Added Official Social Media Follow Links (Facebook & Instagram) of Hasura and also corrected the profile link of twitter to direct short link.
This would help people to follow pages of Hasura on all social media platforms.

(fix #<issue-no>)

### CODE REVIEW

**Comment by midhunbiju** [comment]
Is this come under #hacktoberfest tag ?
If yes, please tag hacktoberfest to my PR.

**Comment by midhunbiju** [comment]
Did this come under #hacktoberfest tag ?
If yes, please tag hacktoberfest to my PR.

File: README.md [hunk 1]
```diff
@@ -5,7 +5,9 @@
 
 
 <a href="https://discord.gg/vBPpJkS"><img src="https://img.shields.io/badge/chat-discord-brightgreen.svg?logo=discord&style=flat"></a>
-<a href="https://twitter.com/intent/follow?screen_name=HasuraHQ"><img src="https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo=twitter"></a>
+<a href="https://twitter.com/HasuraHQ"><img src="https://img.shields.io/badge/Follow-HasuraHQ-blue.svg?style=flat&logo=twitter"></a>
```
Comment (marionschleifer) [inline README.md]: I think I'd prefer the old version, as it leads directly to the follow page.

Comment (midhunbiju) [reply README.md]: Why social media badges are not added to the master of README.md ?

**Pull request closed without merging**